static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js

```diff
@@ -0,0 +1,125 @@
+                minArticlesBeforeShowingBanner: 0,
+                userCohort: 'OnlyNonSupporters',
+            },
+        },
+        {
+            id: 'variant1',
+            test: (): void => {},
+            engagementBannerParams: {
+                messageText: `VARIANT NON UK${defaultCopy}`,
+                // buttonCaption?: string, TO be decided with non engineers
+                template: acquisitionsBannerFivTemplate,
+                bannerModifierClass: 'fiv-banner',
+                minArticlesBeforeShowingBanner: 0,
+                userCohort: 'OnlyNonSupporters',
+            },
+        },
+    ],
+};
+
+export const februaryMomentBannerUk: AcquisitionsABTest = {
+    id: 'FebruaryMomentBannerUk',
+    start: '2019-01-01',
+    expiry: '2019-09-30',
+    author: 'John Duffell',
+    description: 'enable engagement banner in UK for the feb moment',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'AV per impression',
+    audienceCriteria: 'All',
+    idealOutcome: 'best AV possible',
+    canRun: () => getSync() === 'GB',
+    showForSensitive: true,
+    campaignId: 'february_moment', // TODO check this matches the one used elsewhere
+    componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
+    variants: [
+        {
+            id: 'control',
+            test: (): void => {}, // banner tests look at the bucket and vary the copy themselves
+            engagementBannerParams: {
+                messageText: `UK${defaultCopy}`,
+                // buttonCaption?: string, TO be decided with non engineers
+                template: acquisitionsBannerFivTemplate,
+                bannerModifierClass: 'fiv-banner',
+                minArticlesBeforeShowingBanner: 0,
+                userCohort: 'OnlyNonSupporters',
+            },
+        },
+    ],
+};
+
+export const februaryMomentBannerThankYou: AcquisitionsABTest = {
+    id: 'FebruaryMomentBannerThankYou',
+    start: '2019-01-01',
+    expiry: '2019-09-30',
+    author: 'John Duffell',
+    description: 'test copy on engagement banner outside US for the feb moment',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'AV per impression',
+    audienceCriteria: 'All',
+    idealOutcome: 'best AV possible',
+    canRun: () => true,
+    showForSensitive: true,
+    campaignId: 'february_moment', // TODO check this matches the one used elsewhere
+    componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
+    variants: [
+        {
+            id: 'control',
+            test: (): void => {}, // banner tests look at the bucket and vary the copy themselves
+            engagementBannerParams: {
+                messageText: thankYouCopy,
+                // buttonCaption?: string, TO be decided with non engineers
+                template: acquisitionsBannerFivTemplate,
+                bannerModifierClass: 'fiv-banner',
+                minArticlesBeforeShowingBanner: 0,
+                userCohort: 'OnlyExistingSupporters',
```

Comment:
at the moment it doesn't put people into this test, because the users get into the earlier tests based on being either in or not in GB and this one isn't looked at.  This will be in another PR